docs/JLBP-15.md

```diff
@@ -1,51 +1,63 @@
-  it has no `<dependencies>` section. It also has `<packaging>pom</packaging>`.
-  - Documentation is available at
-    https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html
-    .
-  - All of the modules of the project should be included. Otherwise, a consumer
-    who depends on an unspecified module must specify and upgrade the version of
-    that module, defeating the purpose of the BOM.
-- The purpose of a BOM is to enable consumers of a library to select a
-  consistent set of versions for artifacts released by a single project. When
-  imported, a BOM operates as a filter. It does not cause any modules to be
-  added as dependencies. Instead, for any dependency in a project's dependency
-  tree, *if* that module appears in the BOM, the version from the BOM is used.
-- If all modules in a project have the same version, use that version for the
-  BOM. Example: grpc-bom v1.15.0 would specify grpc-auth v1.15.0, gprc-core
-  v1.15.0, etc.
-- A BOM is not necessary for projects that only have one module, since there is
-  no consistency problem in that case.
-- See the details specific to each build system in the sections below.
-
-### Libraries using Maven
-
-- Create a module containing only the BOM `pom.xml` file and add the BOM module
-  to the parent `<modules>` list.
-- A BOM can inherit from a parent as long as the parent is not part of the build
+A [BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html)
+enables consumers of a library to select
+consistent versions for artifacts included in that library. When
+imported, a BOM operates as a filter. It does not cause any modules to be
+added as dependencies. Instead, for any dependency in a project's dependency
+tree, *if* that module appears in the BOM, the version from the BOM is used.
+
+## The BOM (Bill of Materials)
+
+A BOM is a `pom.xml` file that has `<packaging>pom</packaging>`.
+The `<dependencyManagement>` section lists all the modules of a project.
+The BOM can omit the `<modules>` section and does not have a `<dependencies>`
+section.
+
+Consumers of a multi-module library import the library's BOM
+in their own `<dependencyManagement>` section. They can then omit the
+versions from the specific modules they import. The versions will
+be set by the BOM. This provides consistent versions for artifacts released
+by a single project.
+
+Your project's BOM should include every non-test-scoped module in the project.
+Otherwise, a consumer who depends on an unspecified module must specify and
+upgrade the version of that module, defeating the purpose of the BOM.
+
+A BOM is not necessary for projects that only have one module, since there is
+no consistency problem in that case.
+
+See the details specific to each build system in the following sections.
+
+## Libraries using Maven
+
+Create a module containing only the BOM `pom.xml` file and add the BOM module
+to the parent `<modules>` list.
```

Comment:
I would combine this paragraph with the following one, and also merge the two bullet points into the paragraph as well, leaving only the example BOM line on its own.

Reply:
done